src/plugins/unified_histogram/public/layout/hooks/use_lens_suggestions.ts

```diff
@@ -142,6 +144,7 @@ export const useLensSuggestions = ({
     currentSuggestion: histogramSuggestion ?? currentSuggestion,
     suggestionUnsupported: !currentSuggestion && !histogramSuggestion && isPlainRecord,
     isOnHistogramMode: Boolean(histogramSuggestion),
+    histogramQuery: histogramQuery.current ? { esql: histogramQuery.current } : undefined,
```

Comment:
ℹ️  This is done to communicate correctly the histogram query (as it is not the same that Discover uses aka the unified search query)

Reply:
Thanks for the note 👍 Did it not work correctly before or do we just need to do this now because of the changes in this PR?

Reply:
It did not work correctly before but it was not visible to the users as we didnt allow the editing of the query. So in 8.11 if the user saves the histogram to the dashboard they can edit it correctly but the query that is saved with the SO is not the correct one.